dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyTreeFormatter.java

```diff
@@ -98,14 +98,15 @@ private static void formatDependencyPathTree(
   }
 
   /**
-   * Builds ListMultiMap that represents a Maven dependency tree of parent-children relationship.
-   * The root node is represented as an empty {@link DependencyPath} and its children are
-   * the values for the root node.
+   * Builds ListMultimap that represents a Maven dependency tree of parent-children relationship.
    *
    * @param dependencyPaths dependency path instances without assuming any order
-   * @return ListMultiMap representing a Maven dependency tree
+   * @return ListMultimap representing a Maven dependency tree of parent-children relationship. The
+   *     {@link DependencyPath} representing the root Maven artifact is available by querying the
```

Comment:
Multimap keys are nullable. Can we use null as a key here instead? I'm not sure but it's worth trying.

Reply:
First element in values of LinkedListMultimap worked good.

(Null as key would work too in the map in Java. Freemarker is not good at handling null though.)